bot.py

```diff
@@ -467,10 +468,17 @@ async def show_queue(self, message, _command_content):
 
         await message.channel.send(reply)
 
+    async def move(self, message, _command_content):
+        if not self.voice_client:
+            return
```

Comment:
Maybe use `self.create_or_get_voice_client`, or something for user friendly feedback if the user is not in a voice channel.

What should happen if the bot is already in your voice channel?

Reply:
It's fixed now :) I just jumped the gun a bit on the push 😅

Reply:
ahh no sorry, now I understand what you mean with the first part. I'll fix that now!